<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QWidget" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="btn_ordering">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(135, 125, 120);
        color: #fff;
        font-size: 13px;
        font-weight: bold;
        border-top-right-radius: 15px;
        border-top-left-radius: 0px;
        border-bottom-right-radius: 0px;
        border-bottom-left-radius: 15px;
        padding: 10px;

}</string>
   </property>
   <property name="text">
    <string>Odering List</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>641</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="page_0_orders">
    <widget class="QTableView" name="tableView_orders">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>541</width>
       <height>391</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Customer Orders</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_1">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Customer Information</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_customer">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>501</width>
       <height>351</height>
      </rect>
     </property>
     <property name="autoScrollMargin">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QPushButton" name="btn_create_item">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>470</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Item</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>580</width>
       <height>395</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>405</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="itemManagementGrid"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>All Items</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3_add">
    <widget class="QPushButton" name="btn_add_item">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to Menu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editItemText">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Item Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Item Image Path</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>:/resource/images/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editItemImage">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>140</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Item Price</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editItemPrice">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>210</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>$</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>or ./exeImages/ (for running released exe)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4_order_details">
    <widget class="QLabel" name="labelOrderDetail">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_orderDetails">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>541</width>
       <height>331</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ready">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ready</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCode">
     <property name="geometry">
      <rect>
       <x>422</x>
       <y>400</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_customers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(135, 125, 120);
        color: #fff;
        font-size: 13px;
        font-weight: bold;
        border-top-right-radius: 15px;
        border-top-left-radius: 0px;
        border-bottom-right-radius: 0px;
        border-bottom-left-radius: 15px;
        padding: 10px;

}</string>
   </property>
   <property name="text">
    <string>Customers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_staff">
   <property name="geometry">
    <rect>
     <x>712</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_menu_item">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(135, 125, 120);
        color: #fff;
        font-size: 13px;
        font-weight: bold;
        border-top-right-radius: 15px;
        border-top-left-radius: 0px;
        border-bottom-right-radius: 0px;
        border-bottom-left-radius: 15px;
        padding: 10px;

}</string>
   </property>
   <property name="text">
    <string>Menu Items</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
